hatch.toml

```diff
@@ -473,4 +473,10 @@ test = [
 
 [[envs.selenium.matrix]]
 python = ["3.7", "3.10", "3.12"]
-selenium = ["~=3.0", "~=4.0"]
+tested_pytest_plugin_version = ["v1", "v2"]
+
+[envs.selenium.overrides]
+matrix.tested_pytest_plugin_version.env-vars = [
+    { key = "_TESTED_PYTEST_PLUGIN_VERSION", value = "false", if = ["v1"]},
```

Comment:
Can't we just set `DD_PYTEST_USE_NEW_PLUGIN_BETA` directly here? It seems weird to have a "version" variable whose values are true and false. :thinking:

Reply:
Setting `DD_PYTEST_USE_NEW_PLUGIN_BETA` would set the version of the plugin that's running in the `hatch` environment, but we want to affect the version of the plugin that's being tested, and since it's being tested in a subprocess, it needs to be passed in to that subprocess' env.

Since we're running the tests with `--no-ddtrace`, I was worried that using `DD_PYTEST_USE_NEW_PLUGIN_BETA` would end up being confusing.